applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -180,7 +180,8 @@ Testing
 After programming the application and all the prerequisites to your development kit, test the application by performing the following steps:
 
 1. |connect_kit|
-#. Connect to the kit with a terminal emulator (for example, LTE Link Monitor). See :ref:`lte_connect` for more information.
+#. Connect to the kit with a terminal emulator (for example, `nRF Connect Serial Terminal`_).
+   You can also use the :guilabel:`Open Serial Terminal` option of the `Cellular Monitor`_ app to open the Serial Terminal.
```

Comment:
This could be "button" instead of "option", but I wouldn't say it's wrong as it is.